src/control-plane/backend/lambda/api/test/api/metadata.test.ts

```diff
@@ -21,321 +22,340 @@ import {
-          description: `Description of event parameter ${MOCK_EVENT_PARAMETER_NAME}`,
-        },
-        {
-          id: `EVENT_PARAMETER#${MOCK_PROJECT_ID}#${MOCK_APP_ID}#${MOCK_EVENT_PARAMETER_NAME}11`,
-          projectId: MOCK_PROJECT_ID,
-          appId: MOCK_APP_ID,
-          displayName: `display name of event parameter ${MOCK_EVENT_PARAMETER_NAME}11`,
-          description: `Description of event parameter ${MOCK_EVENT_PARAMETER_NAME}11`,
-        },
-        {
-          id: `EVENT_PARAMETER#${MOCK_PROJECT_ID}#${MOCK_APP_ID}#${MOCK_EVENT_PARAMETER_NAME}12`,
-          projectId: MOCK_PROJECT_ID,
-          appId: MOCK_APP_ID,
-          displayName: `display name of event parameter ${MOCK_EVENT_PARAMETER_NAME}12`,
-          description: `Description of event parameter ${MOCK_EVENT_PARAMETER_NAME}12`,
-        },
-        {
-          id: `DICTIONARY#${MOCK_PROJECT_ID}#${MOCK_APP_ID}#${MOCK_EVENT_PARAMETER_NAME}#value-02`,
-          projectId: MOCK_PROJECT_ID,
-          appId: MOCK_APP_ID,
-          displayName: `display name of dictionary ${MOCK_EVENT_PARAMETER_NAME} value-02`,
-        },
-        {
-          id: `DICTIONARY#${MOCK_PROJECT_ID}#${MOCK_APP_ID}#${MOCK_EVENT_PARAMETER_NAME}11#value-02`,
-          projectId: MOCK_PROJECT_ID,
-          appId: MOCK_APP_ID,
-          displayName: `display name of dictionary ${MOCK_EVENT_PARAMETER_NAME}11 value-02`,
-        },
-        {
-          id: `DICTIONARY#${MOCK_PROJECT_ID}#${MOCK_APP_ID}#${MOCK_EVENT_PARAMETER_NAME}12#value-012`,
-          projectId: MOCK_PROJECT_ID,
-          appId: MOCK_APP_ID,
-          displayName: `display name of dictionary ${MOCK_EVENT_PARAMETER_NAME}12 value-012`,
-        },
-        {
-          id: `USER_ATTRIBUTE#${MOCK_PROJECT_ID}#${MOCK_APP_ID}#${MOCK_USER_ATTRIBUTE_NAME}`,
-          projectId: MOCK_PROJECT_ID,
-          appId: MOCK_APP_ID,
-          displayName: `display name of user parameter ${MOCK_USER_ATTRIBUTE_NAME}`,
-          description: `Description of user parameter ${MOCK_USER_ATTRIBUTE_NAME}`,
-        },
-        {
-          id: `DICTIONARY#${MOCK_PROJECT_ID}#${MOCK_APP_ID}#${MOCK_USER_ATTRIBUTE_NAME}#value-02`,
-          projectId: MOCK_PROJECT_ID,
-          appId: MOCK_APP_ID,
-          displayName: `display name of dictionary ${MOCK_USER_ATTRIBUTE_NAME} value-02`,
-        },
-      ],
-    });
-  });
-  it('Create metadata event', async () => {
-    tokenMock(ddbMock, false);
-    ddbMock.on(PutCommand).resolvesOnce({});
-    const res = await request(app)
-      .post('/api/metadata/event')
-      .set('X-Click-Stream-Request-Id', MOCK_TOKEN)
-      .send({
+    ],
+    valueType: 'String',
+  },
+  summary: {
+    category: 'event',
+    eventName: MOCK_EVENT_NAME,
+    hasData: true,
+    name: MOCK_EVENT_PARAMETER_NAME,
+    platform: [
+      'Android',
+      'iOS',
+    ],
+    valueEnum: [
+      {
+        count: 103,
+        value: 'value-01',
```

Comment:
the value is `0` in day1,

```
  day1: {
    category: 'event',
    eventName: MOCK_EVENT_NAME,
    hasData: true,
    name: MOCK_EVENT_PARAMETER_NAME,
    platform: [
      'Android',
    ],
    valueEnum: [
      {
        count: 103,
        value: '0',
      },
    ],
    valueType: 'String',
  },
```

why it became `value-01` here? Does it test the dirty data will be overwritten?

Reply:
0 and 1 are dirty data, fixed.